curriculum/challenges/english/20-upcoming-python/learn-regular-expressions-by-building-a-password-generator/657732654845d8e2fb1217e6.md

```diff
@@ -0,0 +1,68 @@
+---
+id: 657732654845d8e2fb1217e6
+title: Step 62
+challengeType: 20
+dashedName: step-62
+---
+
+# --description--
+
+The `all()` function returns `True` if all the elements inside a given iterable evaluate to `True`. Otherwise it returns `False`.
```

Comment:
```suggestion
`all()` is a built-in Python function that returns `True` if all the elements inside a given iterable evaluate to `True`. Otherwise, it returns `False`.
```